From bf276caacc7b343f7015e4454f982098edaade5c Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Tue, 12 May 2015 10:30:32 -0700
Subject: [PATCH 2433/2848] HADOOP-11962. Sasl message with MD5 challenge text
 shouldn't be LOG out even in debug level.
 Contributed by Junping Du.

(cherry picked from commit 2f4b6d1157f280c8a6e1b2e7217fd2ec16991985)

Change-Id: Iec2b0215be338c33613d2cf080bcffde5aecc743
---
 .../main/java/org/apache/hadoop/ipc/Server.java    |    3 ---
 .../org/apache/hadoop/security/SaslRpcClient.java  |    3 ---
 .../hadoop/security/UserGroupInformation.java      |    3 ---
 3 files changed, 9 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java
index 8d6e9942..bdb2d44 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java
@@ -1562,9 +1562,6 @@ private RpcSaslProto buildSaslResponse(SaslState state, byte[] replyToken) {
     }
 
     private void doSaslReply(Message message) throws IOException {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("Sending sasl message "+message);
-      }
       final Call saslCall = new Call(AuthProtocol.SASL.callId,
           RpcConstants.INVALID_RETRY_COUNT, null, this);
       final ByteArrayOutputStream saslResponse = new ByteArrayOutputStream();
diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslRpcClient.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslRpcClient.java
index 22437b2..919aded 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslRpcClient.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SaslRpcClient.java
@@ -388,9 +388,6 @@ public AuthMethod saslConnect(InputStream inS, OutputStream outS)
       }
       RpcSaslProto saslMessage =
           RpcSaslProto.parseFrom(responseWrapper.getMessageBytes());
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("Received SASL message "+saslMessage);
-      }
       // handle sasl negotiation process
       RpcSaslProto.Builder response = null;
       switch (saslMessage.getState()) {
diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
index 4769641..a5e6cd2 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java
@@ -963,9 +963,6 @@ private synchronized KerberosTicket getTGT() {
         .getPrivateCredentials(KerberosTicket.class);
     for (KerberosTicket ticket : tickets) {
       if (SecurityUtil.isOriginalTGT(ticket)) {
-        if (LOG.isDebugEnabled()) {
-          LOG.debug("Found tgt " + ticket);
-        }
         return ticket;
       }
     }
-- 
1.7.9.5

